docs/user-guide/ze-usage-tips.md

```diff
@@ -3,27 +3,35 @@
 Make the best use of Zowe Explorer with the following tips.
 
 ## Data sets, USS, and jobs persistence settings
+
 You can store any data sets, USS files, or jobs permanently in the **Favorites** tab. Right-click on a data set, USS file, or job and click **Add Favorite**.
 
 ## Identify syntax errors with a syntax highlighter
+
 Zowe Explorer supports a syntax highlighter for data sets. To enhance the experience of using the extension, you can download an extension that highlights syntax.
 
 ## Configure the detected language of a file or data set
 
 You can configure Visual Studio Code to use a specific language for a particular file extension type. This prevents the language for a file or data set opened in Zowe Explorer to be detected incorrectly. To set file associations, see [Add a file extension to a language](https://code.visualstudio.com/docs/languages/overview#_add-a-file-extension-to-a-language).
```

Comment:
This sentence is awkward:
This prevents the language for a file or data set opened in Zowe Explorer to be detected incorrectly.

How about:
This prevents the language for a file or data set opened in Zowe Explorer from being detected incorrectly.